### ISSUE TITLE
[cards] Repeated links to same destination within cards

### ISSUE THREAD

**Issue by UsmanAfzal** [WeAreSnook/OR-UK-frontend#179]
URL: https://openreferraluk.org/standard-community

There are repeated links to the same destination within each card one from the logo and the other from the text "Visit their website". The logo links should be removed as they are not suitable for screenreaders as they do not give helpful information. 

Example: 

<img width="763" alt="Screenshot 2021-06-28 at 13 36 24" src="https://user-images.githubusercontent.com/2459151/123637515-033ad700-d816-11eb-827d-a208293490ac.png">

Current HTML:
```
<a href="https://www.adur-worthing.gov.uk/A" class="card__img-container link-with-image">
  <div class="CompanyLogo" title="company_logo" style="background-image: url(&quot;https://admin.beta.openreferraluk.org/uploads/thumbnail_adur_worthing_05d007358d.png&quot;);"></div>
</a>
```

Change to:
```
<div class="card__img-container">
  <div class="CompanyLogo" style="background-image: url(&quot;https://admin.beta.openreferraluk.org/uploads/thumbnail_adur_worthing_05d007358d.png&quot;);"></div>
</div>
```

**Comment by kingdomlevel** [comment]
PR #209 has now been merged - removing the 'blocked' label and moving to 'To Do'

**Comment by UsmanAfzal** [comment]
@adamgparsons @kingdomlevel The title attribute was also meant to be removed here.

**Comment by UsmanAfzal** [comment]
Checked and approved
